indico/core/settings/converters.py

```diff
@@ -144,3 +146,13 @@ def to_python(self, value):
         if not value:
             return []
         return self.collection_class(self.model.query.filter(self.column.in_(value)))
+
+
+class OrderedDictConverter(SettingConverter):
+    @staticmethod
+    def from_python(value):
+        return json.dumps(value)
```

Comment:
I really don't like the idea of storing nested JSON... I'd rather convert it to a list of lists...

Reply:
So, does it really hurt? By having an actual `dict` here the `update/delete` operations are much easier than iterating over the entire list of lists

Reply:
i mean for storage - nothing wrong with the dict itself

Reply:
Ah.. I see